### PR TITLE
Increase base fee from 12.5% higher than last block to 33%

### PR DESCRIPTION
**Pull request by bbondy**
Resolves https://github.com/brave/brave-browser/issues/22640

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/brave_wallet/browser/eth_gas_utils.cc [hunk 1]
```diff
@@ -64,7 +64,7 @@ bool GetSuggested1559Fees(const std::vector<std::string>& base_fee_per_gas,
   // fee. So the excess will be refunded and it will be at most 12.5% It's best
```
Comment (onyb) [inline components/brave_wallet/browser/eth_gas_utils.cc:64]: Should be 33%

**Comment by onyb** [comment]
## Recording behaviour relative to Etherscan (at random instances)

### Before this PR

|    Brave Wallet  | Etherscan      | % variance |
|----------------|--------------|---|
| Base: 116 | Base: 104 | 11% |
| Base: 114 | Base: 96 | 18% |
| Base: 104 | Base: 93 | 11%
| Base: 90 | Base: 81 | 11% |
| Base: 87 | Base: 78 | 11% |

### After this PR

|    Brave Wallet  | Etherscan      | % variance |
|----------------|--------------|---|
| Base: 95 | Base: 76 | 25% |
| Base: 107 | Base: 80 | 33% |
| Base: 126 | Base: 99 | 27% |
| Base: 119 | Base: 89 | 33% |
| Base: 123 | Base: 90 | 36% |

**Review by onyb — APPROVED** [review]
Left some comparisons w/ Etherscan to get an idea of how base fee is going to change.

**Comment by bbondy** [comment]
> Left a comment [here](https://github.com/brave/brave-browser/issues/22640) about this. Based on this PR, I think you internally use `base fee` to mean `max fee`, but I haven't looked too deeply. Just want to flag!

Internally we mean it to be max fee minus the priority fee. I agree the naming is not ideal, we'll clean that up in a future commit.

**Comment by timbeiko** [comment]
Ok cool - just wanted to make sure it didn't break anything unexpectedly :-) !

**Pull request merged**